spring-integration-hazelcast/src/test/resources/HazelcastCQDistributedMapInboundChannelAdapterTests-context.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns:int="http://www.springframework.org/schema/integration"
+  xmlns:int-hazelcast="http://www.springframework.org/schema/integration/hazelcast"
+  xsi:schemaLocation="http://www.springframework.org/schema/beans
+    http://www.springframework.org/schema/beans/spring-beans.xsd
+    http://www.springframework.org/schema/integration
+    http://www.springframework.org/schema/integration/spring-integration.xsd
+    http://www.springframework.org/schema/integration/hazelcast
+	classpath:org/springframework/integration/hazelcast/config/xml/spring-integration-hazelcast-4.2.xsd">

```

Comment:
??? 
Why `classpath`?
It really must be like all other XSD locations:
```
http://www.springframework.org/schema/integration/hazelcast
    http://www.springframework.org/schema/integration/hazelcast/spring-integration-hazelcast.xsd
```
And, please, always use `versionless` xsd locations